src/main/java/de/tum/in/www1/artemis/service/user/UserCreationService.java

```diff
@@ -252,6 +252,31 @@ public User updateUser(@NotNull User user, ManagedUserVM updatedUserDTO) {
         return saveUser(user);
     }
 
+    /**
+     * Update all information for a specific user (including its password) based on the data made available by the LDAP
```

Comment:
This comment is wrong. The password is not synchronized to Artemis. Please change it